Octokit.Tests.Integration/Clients/IssuesClientTests.cs

```diff
@@ -61,6 +61,25 @@ public IssuesClientTests()
     }
 
     [IntegrationTest]
+    public async Task CanLockAndUnlockIssue()
+    {
+        var newIssue = new NewIssue("a test issue") { Body = "A new unassigned issue" };
+        var issue = await _issuesClient.Create(_context.RepositoryOwner, _context.RepositoryName, newIssue);
+        var retrieved = await _issuesClient.Get(_context.RepositoryOwner, _context.RepositoryName, issue.Number);
+        Assert.False(issue.Locked);
+ 
+        await _issuesClient.Lock(_context.RepositoryOwner, _context.RepositoryName, issue.Number);
+        retrieved = await _issuesClient.Get(_context.RepositoryOwner, _context.RepositoryName, issue.Number);
```

Comment:
Actually, you should be returned the updated `Issue` from `Lock` and `Unlock` here - to simplify this test (and do less network calls) you could assign this returned value to `retrieved`...